### ISSUE TITLE
photos don't show when click on them - wrong link

### ISSUE THREAD

**Issue by spirogg** [MacWarrior/clipbucket-5.0#6]
photos don't show when click on them - wrong link

```
non seo = https://domain.com/view_item.php?item=X3KWBX3M3RB8&type=photos&collection=10
wrong link

SEO = https://domain.com/item/photos/10/X3KWBX3M3RB8/Social-network
wrong link

if I view source for image it shows this = https://domain.com/photos/2018/05/26/1527375889f5c06c.png

but should be this = https://domain.com/files/photos/2018/05/26/1527375889f5c06c.png
```

**Comment by MacWarrior** [comment]
Please update to latest revision ; this [commit ](https://github.com/MacWarrior/clipbucket-5.0/commit/502b2bd642a6b0a559485fcac79a8d455dfbc69b) may correct the links

**Comment by MacWarrior** [comment]
First time I use photos on CB ; I have discovered multiples errors ; will work on it

**Comment by spirogg** [comment]
ok still I think 1 issue when click on collections 

photos seem to take video instead and dont show see photos

![collection1](https://user-images.githubusercontent.com/17736688/40588978-7555546c-61ab-11e8-93c7-45f0c9137b22.PNG)
![collections](https://user-images.githubusercontent.com/17736688/40588979-7562d66e-61ab-11e8-8b47-f0ff9d3f0a3c.PNG)
![top collections](https://user-images.githubusercontent.com/17736688/40588980-756d7812-61ab-11e8-9c0d-97c9b57d5e9b.PNG)

**Comment by spirogg** [comment]
```
<?php
	/*
	 ******************************************************************
	 | Copyright (c) 2007-2008 Clip-Bucket.com. All rights reserved.
	 | @ Author : ArslanHassan
	 | @ Software : ClipBucket , © PHPBucket.com
	 ******************************************************************
	*/
	define("THIS_PAGE",'view_collection');
	define("PARENT_PAGE",'collections');

	require 'includes/config.inc.php';
	$userquery->perm_check('view_video',true);
	$pages->page_redir();

	$c = mysql_clean((int)$_GET['cid']);
	$type = mysql_clean($_GET['type']);

	$page = mysql_clean($_GET['page']);
	$get_limit = create_query_limit($page,COLLIP);
	$order = tbl("collection_items").".ci_id DESC";

	if($cbcollection->is_viewable($c))
	{
		$param = array("type"=>$type,"cid"=>$c);
		$cdetails = $cbcollection->get_collection($c,"AND ".tbl($cbcollection->section_tbl).".type = '$type' ");

		if($cdetails)
		{
		switch($type)
		{
			case "videos":
			case "video":
			case "v":
				$items = $cbvideo->collection->get_collection_items_with_details($c,$order,$get_limit);
				$count = $cbvideo->collection->get_collection_items_with_details($c,NULL,NULL,TRUE);
				break;

			case "photos":
			case "photo":
			case "p":
				if (isSectionEnabled('photos')) {
					$items = $cbphoto->collection->get_collection_items_with_details($c,$order,$get_limit);
					$count = $cbphoto->collection->get_collection_items_with_details($c,NULL,NULL,TRUE);
				} else {
					$Cbucket->show_page = false;
				}
				break;
		}

		// Calling necessary function for view collection
		call_view_collection_functions($cdetails);

		$total_pages = count_pages($count,COLLIP);
		//Pagination
		//$pages->paginate($total_pages,$page);
		$link==NULL;
		$extra_params=NULL;
		$tag='<li><a #params#>#page#</a><li>';
		$pages->paginate($total_pages,$page,$link,$extra_params,$tag);

		assign('objects',$items);
		assign("c",$cdetails);
		assign("type",$type);
		assign("cid",$c);
		subtitle($cdetails['collection_name']);
		} else {
			e(lang("collection_not_exists"));
			$Cbucket->show_page = false;
		}
	} else {
		$Cbucket->show_page = false;
	}

	//Getting Collection List
	$page = mysql_clean($_GET['page']);
	$get_limit = create_query_limit($page,COLLPP);      <<<<<<<<<<<COLLIP ??
	$clist = $cond;
	$clist['limit'] = $get_limit;
	$collections = $cbcollection->get_collections($clist);

	assign('collections', $collections);
	template_files('view_collection.html');
	display_it();
```

**Comment by spirogg** [comment]
Hi MacWarrior, hope your doing well :)

**Comment by MacWarrior** [comment]
Hi, 
This may have been fixed in this commit : 4047a6daf74ec2bdc4cc54e27e5ce19a5acc4af3
